sessions/sessions_test10.c

```diff
@@ -21,26 +30,26 @@ int main (int argc, char *argv[])
 
     rc = MPI_Session_create_errhandler (my_session_errhandler, &errhandler);
     if (MPI_SUCCESS != rc) {
-        fprintf (stderr, "Error handler creation failed with rc = %d\n", rc);
-        abort ();
+        print_error("Error handler creation failed with rc = %d\n", rc);
```

Comment:
This is okay but its not going to do what you think.  The print_error function is not very smart, it only takes a string and the error code.  So your rc = %d\n" will actually show up as rc = %d in the output I think.  probably easiest would be to remove all the with rc = %d\n from the string arg to print_error.